### DISCUSSION TITLE
Using Azure Bot Service / Bot Framework SDK in conjunction with Semantic Kernel

### DISCUSSION THREAD

**Discussion by dluc** [microsoft/semantic-kernel#6629] [Q&A] [answered]
from "thebaconvenger" on Discord:

> Is anyone using Azure Bot Service / Bot Framework SDK in conjunction with Semantic Kernel?  I want to make my Copilot be accessible via Web, Slack, and Email to start so I thought it would be a good fit.  I don't often see it included in samples & recommendations so I'm wondering if look at it would be.a mistake for building a Copilot?

> I mostly wanted to ask because some of the repos look like they haven't been touched in a long time and wasn't sure if its considered stable or potentially being no longer maintained in favor of some other solution that I might not be aware of

src: https://discord.com/channels/1063152441819942922/1075907964667437072/1248785904613589044

**Comment by sophialagerkranspandey** [comment, marked as answer]
Here's a sample showcasing what you're looking for: https://github.com/Azure/AI-in-a-Box/tree/main/gen-ai/semantic-kernel-bot-in-a-box
